PWGGA: Add Helper Class for PHOS Trigger Mimicking

jluehder opened the pull request:
Adding a new Task, which checks events if they are triggered.
Also changed run ranges in AliPHOSTriggerUtils.cxx.
Changes to Trigger Mimick function, to use this class in PHOS

alibuild commented:
9ee2fbf63653f4bbf7ee2d4dd86e45210b5fee8a: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

kharlov commented:
+1

alibuild commented:
9ee2fbf63653f4bbf7ee2d4dd86e45210b5fee8a: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

nschmidtALICE commented on PWGGA/GammaConvBase/AliConvEventCuts.cxx:
```diff
@@ -5041,6 +5044,16 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
+      return kFALSE;
+    }
+  }
+
```
If this is the only way that trigger mimicking should work for PHOS, then please make it an AliFatal if the helper is not found.

nschmidtALICE commented on PWGGA/GammaConvBase/AliConvEventCuts.h:
```diff
@@ -760,6 +765,7 @@ class AliConvEventCuts : public AliAnalysisCuts {
       TString                     fNameHistoReweightingMultMC;            ///< Histogram name for reweighting Eta
       TH1D*                       hReweightMultData;                      ///< histogram input for reweighting Eta
       TH1D*                       hReweightMultMC;                        ///< histogram input for reweighting Pi0
+      phosTriggerType             fPHOSTrigger;                           // Kind of PHOS trigger: L0,L1
```
I guess the classdef needs increasing.

alibuild commented:
ed5a5b732f1a2e8d0b3908a05ebdbcf3b5c57d88: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

jluehder commented:
I did change some minor things:
1) I changed to the correct author
2) I changed the name Mimick to Mimic, for correct english
3) I increades the Class Def in AliConvEventCuts

alibuild commented:
a7cfbe5beea9c64cdc91c3b3cc46bc33e1da91d0: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

jluehder commented:
I added AliFatals
For the new Mimic method the case fMimicTrigger == 3 is now needed

nschmidtALICE commented:
+test

alibuild commented:
a7cfbe5beea9c64cdc91c3b3cc46bc33e1da91d0: testing approved: will not be automatically merged; starting testing. If testing succeeds, merging will require further approval from 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

alibuild commented:
Error while checking build/AliPhysics/release for a7cfbe5beea9c64cdc91c3b3cc46bc33e1da91d0 at 2020-06-03 09:59:
```
sw/BUILD/AliPhysics-latest/log
 EmcHit::EmcHit()
 ^~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'void* ROOTDict::new_AliCaloTriggerMimicHelper(void*)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6273:28: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return  p ? new(p) ::AliCaloTriggerMimicHelper : new ::AliCaloTriggerMimicHelper;
                            ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6273:62: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return  p ? new(p) ::AliCaloTriggerMimicHelper : new ::AliCaloTriggerMimicHelper;
                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'void* ROOTDict::newArray_AliCaloTriggerMimicHelper(Long_t, void*)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6276:62: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return p ? new(p) ::AliCaloTriggerMimicHelper[nElements] : new ::AliCaloTriggerMimicHelper[nElements];
                                                              ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6276:107: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return p ? new(p) ::AliCaloTriggerMimicHelper[nElements] : new ::AliCaloTriggerMimicHelper[nElements];
                                                                                                           ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'int G__G__PWGGAPHOSTasks_649_0_1(G__value*, const char*, G__param*, int)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:13371:108: error: call of overloaded 'AliCaloTriggerMimicHelper(const char*, Int_t)' is ambiguous
        p = new AliCaloTriggerMimicHelper((const char*) G__int(libp->para[0]), (Int_t) G__int(libp->para[1]));
                                                                                                            ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:13373:121: error: call of overloaded 'AliCaloTriggerMimicHelper(const char*, Int_t)' is ambiguous
        p = new((void*) gvp) AliCaloTriggerMimicHelper((const char*) G__int(libp->para[0]), (Int_t) G__int(libp->para[1]));
                                                                                                                         ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/alisw/AliPhysics/14498/a7cfbe5beea9c64cdc91c3b3cc46bc33e1da91d0/build_AliPhysics_release/fullLog.txt).

alibuild commented:
9fdf74c95118adda218abf06a3e59d206a9e4a70: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

nschmidtALICE commented:
+test

alibuild commented:
9fdf74c95118adda218abf06a3e59d206a9e4a70: testing approved: will not be automatically merged; starting testing. If testing succeeds, merging will require further approval from 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

alibuild commented:
Error while checking build/AliPhysics/release for 9fdf74c95118adda218abf06a3e59d206a9e4a70 at 2020-06-03 11:58:
```
sw/BUILD/AliPhysics-latest/log
[ 27%] Building CXX object PWGGA/EMCALTasks/CMakeFiles/PWGGAEMCALTasks-object.dir/AliAnalysisTaskEMCALPhotonIsolation.cxx.o
[ 27%] Building CXX object PWGGA/PHOSTasks/CMakeFiles/PWGGAPHOSTasks-object.dir/G__PWGGAPHOSTasks.cxx.o
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'void* ROOTDict::new_AliCaloTriggerMimicHelper(void*)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6273:28: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return  p ? new(p) ::AliCaloTriggerMimicHelper : new ::AliCaloTriggerMimicHelper;
                            ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6273:62: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return  p ? new(p) ::AliCaloTriggerMimicHelper : new ::AliCaloTriggerMimicHelper;
                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'void* ROOTDict::newArray_AliCaloTriggerMimicHelper(Long_t, void*)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6276:62: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return p ? new(p) ::AliCaloTriggerMimicHelper[nElements] : new ::AliCaloTriggerMimicHelper[nElements];
                                                              ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:6276:107: error: call of overloaded 'AliCaloTriggerMimicHelper()' is ambiguous
       return p ? new(p) ::AliCaloTriggerMimicHelper[nElements] : new ::AliCaloTriggerMimicHelper[nElements];
                                                                                                           ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx: In function 'int G__G__PWGGAPHOSTasks_649_0_1(G__value*, const char*, G__param*, int)':
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:13371:108: error: call of overloaded 'AliCaloTriggerMimicHelper(const char*, Int_t)' is ambiguous
        p = new AliCaloTriggerMimicHelper((const char*) G__int(libp->para[0]), (Int_t) G__int(libp->para[1]));
                                                                                                            ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
     AliCaloTriggerMimicHelper(const char *name="AliCaloTriggerMimicHelper", Int_t clusterType = 0, Bool_t isMC=kFALSE);
     ^~~~~~~~~~~~~~~~~~~~~~~~~
/mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.cxx:13373:121: error: call of overloaded 'AliCaloTriggerMimicHelper(const char*, Int_t)' is ambiguous
        p = new((void*) gvp) AliCaloTriggerMimicHelper((const char*) G__int(libp->para[0]), (Int_t) G__int(libp->para[1]));
                                                                                                                         ^
In file included from /mnt/mesos/sandbox/sandbox/sw/BUILD/66d028bc491d7e4938b544e77927ed57a1f97c36/AliPhysics/PWGGA/PHOSTasks/G__PWGGAPHOSTasks.h:39:0,
--
/mnt/mesos/sandbox/sandbox/sw/SOURCES/AliPhysics/build_AliPhysics_release/0/PWGGA/PHOSTasks/ClusterSelection/AliCaloTriggerMimicHelper.h:16:5: note: candidate: AliCaloTriggerMimicHelper::AliCaloTriggerMimicHelper(const char*, Int_t, Bool_t)
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/alisw/AliPhysics/14498/9fdf74c95118adda218abf06a3e59d206a9e4a70/build_AliPhysics_release/fullLog.txt).

alibuild commented:
217d4621b4beb8bd5f929323d336d8f7d072753e: approval required: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to approve and allow automatic merging,or with `+test` to run tests only. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

nschmidtALICE commented:
+test

alibuild commented:
217d4621b4beb8bd5f929323d336d8f7d072753e: testing approved: will not be automatically merged; starting testing. If testing succeeds, merging will require further approval from 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

alibuild commented:
217d4621b4beb8bd5f929323d336d8f7d072753e: tests OK, approval required for merging: 1 of @davidrohr (David Rohr), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan); 1 of @davidrohr (David Rohr), @loizides (Constantinos Loizides), @gconesab (Gustavo Conesa Balbastre), @jgrosseo (Jan Fiete Grosse-Oetringhaus), @qgp (Jochen Klein), @peressounko (Dmitri Peresunko), @policheh (Boris Polishchuk), @kharlov (Yuri Kharlov), @nschmidtALICE (Nicolas Vincent Schmidt), @dmuhlhei (Daniel Michael Muhlheim), @FriederikeBock (Friederike Bock), @shahor02 (Ruben Shahoyan)

_Comment with `+1` to merge. **Please comment on the pull request: [click here](https://github.com/alisw/AliPhysics/pull/14498) and comment at the bottom of the page.**_

nschmidtALICE commented:
+1
